backend/admins/api/urls.py

```diff
@@ -1,7 +1,11 @@
 from django.urls import path
-from .views import CategoriesTable, SingleCategory
+from .views import CategoriesTable, SingleCategory, CreateQuestionAPI, UpdateQuestionAPI, DeleteQuestionAPI, CategoryQuestions
 
 urlpatterns = [
     path('categories', CategoriesTable.as_view(), name='allCategories'),
     path('category/<int:pk>', SingleCategory.as_view(), name='singleCategory'),
+    path('createQuestion', CreateQuestionAPI.as_view(), name='createQuestion'),
+    path('updateQuestion/<int:pk>', UpdateQuestionAPI.as_view(), name='updateQuestion'),
+    path('deleteQuestion/<int:pk>', DeleteQuestionAPI.as_view(), name='deleteQuestion'),
+    path('categoryQuestions/<int:pk>', CategoryQuestions.as_view(), name='categoryQuestions'),
```

Comment:
I am expecting 1 or 2 url instead of 4-5urls.
You can make it dynamic by using same url but different methods.
Make sure to use the right request methods like GET/POST/PUT/PATCH/DELETE of your FE. 
With the use of the right method, you can know already that this method is for update/delete/etc. without calling the url like /updateQuestion which is not good.

You can get idea here on how to do that.
https://iq.opengenus.org/crud-application-with-django/

Sample of CRUD endpoint best practice:
![image](https://user-images.githubusercontent.com/80734011/193004076-ea2fedab-2200-4e77-b44b-28cab450b1c1.png)